posts/2019/07/ibis-python-data-analysis-productivity-framework.ipynb

```diff
@@ -0,0 +1,424 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Ibis: Python Data Analysis Productivity Framework\n",
+    "\n",
+    "Ibis is a library pretty useful on data analysis tasks. That allows you work locally over a schema, like create filter, add columns, apply math operations etc and when you want to get the result of the expression you created, Ibis compiles that and makes a request to the remote server (remote storage and execution systems like Hadoop components or SQL databases). Its goal is to simplify analytical workflows and make you more productive.\n",
+    "\n",
+    "Ibis was created by Wes McKinney (@wesm) and is mainly maintained by Phillip Cloud (@cpcloud) and Krisztián Szűcs (kszucs). Also, recently, I was invited as contributor to Ibis repository!\n",
+    "\n",
+    "Maybe you are thinking: \"why should I use Ibis?\". Well, if you have any of these follow issues, probably you should consider use ibis in your analytical workflow!\n",
+    "\n",
+    "- if you need to get data from a SQL Database but you don't know much about SQL ...\n",
+    "- if you create SQL statements manually using string and have a lot of `IF`'s in your code that compose specific parts of your SQL code (it could be pretty hard to maintain and it will makes your code pretty ugly) ...\n",
+    "- if you need to handle data with big volume ..."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<!-- TEASER_END -->"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If you want to learn more about ibis consider to take a look into these tutorials:\n",
+    "\n",
+    "* https://docs.ibis-project.org/tutorial.html\n",
+    "\n",
+    "Do you want to watch some interesting video about Ibis? Check this out:\n",
+    "\n",
+    "* https://www.youtube.com/embed/8Tzh42mQjrw?start=1625\n",
+    "\n",
+    "**Now, let's check out some work developed here at Quansight in the last months!**\n",
+    "\n",
+    "During the last months OmniSci and Quansight were working together to add a backend on Ibis for OmniSciDB (former MapD) that also resulted on new expressions/operators on Ibis core, such as:\n",
```

Comment:
Would be good to say in 1-2 sentences what OmniSciDB is, because many readers won't know. E.g. "A GPU-accelerated SQL database, .... "